CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+ - LS abort on LOADB error, PAGE upon success.
+ - RDIR improve placement of MORE.
+ 
+## [2.0.0] - 2020-05-26
```

Comment:
It looks strange, now 2.0.0 is added twice.

Reply:
I don't know. Should the line be removed, incremented or ?